app/models/library.rb

```diff
@@ -1,8 +1,10 @@
 class Library < ActiveRecord::Base
   belongs_to :category, class_name: 'LibraryCategory', foreign_key: "category_id"
+  belongs_to :user
   validates_presence_of :category
+  validates_presence_of :user
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.